### ISSUE TITLE
returning false from 'before' hook event is not stopping the event 

### ISSUE THREAD

**Issue by Mathijs003** [handsontable/angular-handsontable#96]
### Description
returning false from the 'before' hook event is not stopping the event as it should. 

### Steps to reproduce
```
<hot-table
          [colHeaders]="true"
          [rowHeaders]="true"
          [data]="data"
          (beforeRemoveRow)="contextMenuRemove($event)"
          [contextMenu]="['row_above', 'row_below','remove_row','---------','col_left', 'col_right', 'remove_col']"
        ></hot-table>

```

```
contextMenuRemove(event) {
   if (something) {
      // don't do it
      return false; // this is not working
    } else {
      //do it
    }
  }

```

**Comment by Mathijs003** [comment]
@AMBudnik No, because the return false is not stopping the deletion of the row

**Comment by Mathijs003** [comment]
example : [https://stackblitz.com/edit/handsontable-angular-hooks](https://stackblitz.com/edit/handsontable-angular-hooks)

**Comment by Mathijs003** [comment]
example : https://stackblitz.com/edit/handsontable-angular-hooks-bgrip2

**Comment by AMBudnik** [comment]
I've just posted a demo (using Angular) at the second issue.

**Comment by Mathijs003** [comment]
Fixed in [#95](https://github.com/handsontable/angular-handsontable/issues/95)
